### PR TITLE
Adds dependent plans build documentation

### LINKED ISSUE #5123: Local development with project hierarchy is  not possible yet?
Hello!

I created `my/project1` locally. I built it and installed into the system via `sudo hab pkg install`
Now I have `my/project2` which is depends on `my/project1`.
And now I don't see a way/manual/bug how to build `my/project2`.

Probably we need to document that scenario somehow. Its kinda common use-case and really blocker.

Am I missing something?

### PR DESCRIPTION
**Pull request by rsertelon**
> Closes #5123

Signed-off-by: Romain Sertelon <romain@sertelon.fr>

### CODE REVIEW

**Comment by thesentinels** [comment]
Thanks for the pull request! Here is what will happen next:
 1. Your PR will be reviewed by the maintainers
 2. If everything looks good, one of them will `approve` it, and your PR will be merged.

Thank you for contributing!

File: www/source/partials/docs/_dev-pkgs-plan-builds.html.md.erb [hunk 1]
```diff
@@ -57,6 +57,19 @@ Depending on the platform of your host and your Docker configuration, the behavi
 
 > Note: For more details related to Windows containers see [Running Habitat Windows Containers](/docs/best-practices/#running-habitat-windows-containers).
 
+### Building dependent plans in the studio
```
Comment (cnunciato) [inline www/source/partials/docs/_dev-pkgs-plan-builds.html.md.erb]: We should capitalize the S in Studio.

Comment (kagarmoe) [reply www/source/partials/docs/_dev-pkgs-plan-builds.html.md.erb]: Prefer title-case: "Building Dependent Plans in the Studio"

**Comment by rsertelon** [comment]
Comment addressed :)

**Pull request merged**